src/main/java/uz/spiders/propertymanagement/entities/Application.java

```diff
@@ -0,0 +1,31 @@
+package uz.spiders.propertymanagement.entities;
+
+
+import lombok.Data;
+
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+import javax.persistence.MapsId;
+import java.time.LocalDateTime;
+
+@Data
+@Entity
+public class Application {
+
+    @EmbeddedId
+    private UserPropertyCompositeId id;
```

Comment:
Composite Id name is ambiguous, try to modify to another name

Reply:
I think this naming is understandable enough, just shut it.